test/compilable/test19656.d

```diff
@@ -0,0 +1,11 @@
+// PERMUTE_ARGS:
```

Comment:
You could add `// EXTRA_FILES: imports/test19656a.d imports/test19656b.d imports/test19656c.d`

Reply:
Done